test/SlowTests/Issues/RavenDB-15772.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations.Configuration;
+using Raven.Client.ServerWide.Operations;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_15772 : RavenTestBase
+    {
+        public RavenDB_15772(ITestOutputHelper output) : base(output)
```

Comment:
let's create a test for each of the commands

Reply:
Done